src/sdk/models/policyStatistics.ts

```diff
@@ -21,9 +21,16 @@ export interface StatisticValue {
 export interface LocationStatistic extends LabelledStatistic, StatisticValue {
 }
 
+export enum RemediationStatus {
+	NotRemediated = 'not_remediated',
+	Remediated = 'remediated',
+	Mitigated = 'mitigated'
+}
+
 export interface PolicyStatistic extends LabelledStatistic, StatisticValue {
-	not_remediated?: StatisticValue;
```

Comment:
This isn't correct. We don't report a difference between `remediated` and `mitigated` for the statistics.